src/StatusQ/Components/StatusChatToolBar.qml

```diff
@@ -166,7 +168,7 @@ Rectangle {
                 text: "Activity"
                 orientation: StatusToolTip.Orientation.Bottom
                 y: parent.height + 12
-                arrow.x: width - (parent.width-4)
+                cornerToolTip: isCorner
```

Comment:
2 thoughts here:

1. I think `cornerToolTip` is not a good name for this API. Looking at the tooltip standalone, it doesn't make too much sense. Maybe there are even other use cases where the same visuals are desired and we're not dealing with a window "corner".  How about `offset` which lets you specify the actual offset of the tooltip (while the arrow is always centered relative to the trigger?

2. Instead of introducing an `isCorner` API on the toolbar (which again, is very contextually named), how about we alias the tooltip on the `notificationButton` as `tooltip`? Then consumers of `StatusChatToolBar` can do something like:

```qml
StatusChatToolBar {
  notificationButton.tooltip.offset: ...  // notificationButton is already exposed
}
```

This even opens possibilities to other issues, where right now the tooltips aren't i18n'able.

With this change we can then do:

```qml
StatusChatToolBar {
  notificationButton.tooltip.text = "..."
}
```

Using `offset` as described above would also allow users to offset the tooltip in both directions if needed.

Reply:
Oh and lastly: you can then also apply whatever is needed in `DemoApp.qml` so that everything looks natural there (as in, the tooltip doesn't need an offset in the toolbar because it's not super close to the window)

Reply:
Hey @PascalPrecht makes sense the points from you. I was only thinking of corner uses cases. But the StatusToolTip as widget definitely needs a more usable API from the outside. Will rework on this.